pb/summary/summary.proto

```diff
@@ -76,6 +76,19 @@ message TestInfo {
 
   // The flakiness of the test, measured out of 100
   float flakiness = 8;
+
+  enum Trend{
+    NONE = 0;
```

Comment:
nit: I think it's worth differentiating 'unknown' (we didn't set the trend value) and 'no change' (no change in trend occurred)?